### DISCUSSION TITLE
Galaxy brain badge doesn't show

### DISCUSSION THREAD

**Discussion by fdistasio** [community/community#121349] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I answered two questions and was accepted as an answer but did not receive the galaxy brain badge.
I've read that this happens if you answer your own questions but I've never done it. What could it be due to?

**Comment by davevad93** [comment, marked as answer]
Ciao @fdistasio ,  GitHub ha disabilitato  la funzione di ricevere gli "Achievements" attraverso la Community dato che hanno registrato un aumento di risposte di scarsa qualità e a quanto pare, molti utenti hanno abusato di questo sistema semplicemente per ricevere le badge peggiorando di conseguenza l'ecosistema della Community. Ti lascio qui sotto il link di un post ufficiale da parte dello staff di GitHub che spiega dettagliatamente i motivi di questa decisione:

- [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536)

Buona serata!

**Reply by ebndev** [reply]
Hi @davevad93, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

I do see that you've linked to our announcement about [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536), so thanks for that!

**Reply by davevad93** [reply]
Hi @ebndev ,

My bad, I should have red carefully the GitHub code of conduct so I thank you for bringing this to my attention.
You may see another thread here in the Community discussions where I answered a question from a Spanish speaker in Spanish, but that was before I was aware that only English is permitted, so from now on I'll keep that in mind.

Thank you very much!
